src/qt/clientmodel.cpp

```diff
@@ -246,12 +246,12 @@ QString ClientModel::formatClientStartupTime() const
 
 QString ClientModel::dataDir() const
 {
-    return GUIUtil::boostPathToQString(GetDataDir());
+    return GUIUtil::PathToQString(GetDataDir());
```

Comment:
@PastaPastaPasta would you backport-script recognize `scripted-diff: Merge https://github.com/bitcoin-core/gui/pull/523: scripted-diff: Rename func…` as a correct description?

Reply:
I think it will yes